tests/automl/test_automl_image_object_detection_model_wrapper.py

```diff
@@ -125,6 +125,15 @@ def test_wrap_automl_object_detection_model(self):
                 classes=class_names)
             validate_wrapped_object_detection_model(wrapped_model, data)
 
+            # Now test what happens when an invalid classes input is given
+            label_dict = {'can': 1, 'carton': 2,
+                          'milk_bottle': 3, 'water_bottle': 4}
+            with pytest.raises(TypeError):
```

Comment:
why is TypeError checked for here but ValueError is thrown in the code above?

Reply:
Good catch, fixed!